Demos/02-extract/README.md

```diff
@@ -111,6 +111,19 @@ The next step is to use the Azure Data Factory to create a pipeline to extract t
 
     ![Screenshot of the Azure Data Factory menu](./../../Images/adfv2-setup-03.png)
 
+1. [Optional] By default the Azure Data Factory will use an **Integration Runtime** that is auto-resolving the region. As the Microsoft Graph Data Connect requires that your source and destination, and integration runtime to exist in the same Office 365 region, it is recommended that you create a new Integration Runtime with a fixed region.
+    
+    1. Click on *Connections*, at the bottom of the screen, then choose *Integration Runtimes*
```

Comment:
Rewrite to follow MSFT style guidelines (don't use "click / choose", use "select")... things you select should be bold:

At the bottom of the screen, select **Connections** > **Integration Runtimes**.